content/en/about/capabilities.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Jenkins X Capabilities Matrix
+linktitle: Capabilities Matrix
+description: Matrix that describes the current state of Jenkins X capabilities for major Kubernetes platforms
+type: docs
+weight: 20
+---
+
+## Jenkins X Capabilities Matrix
+
+| Capability | GKE | EKS | AKS | OpenShift 3.11 |
+| --- | :---:| :---: | :---: | :---: |
+| **Terraform scripts for cluster creation** | Yes | Yes | No | No |
+| **`jx boot` for installation** | Yes | Yes | No | Yes |
+| **Vault for secret storage** | Yes | Yes | No | Self Provisioned |
+| **Nexus** | Yes | Yes | No | Yes |
```

Comment:
Why is Nexus worth mentioning? That seems to me a bit random and too specific on this level of abstractio.